Tests/MacroToolkitTests/MacroToolkitTests.swift

```diff
@@ -20,70 +20,80 @@ let testMacros: [String: Macro.Type] = [
 ]
 
 final class MacroToolkitTests: XCTestCase {
+    override func invokeTest() {
+        withMacroTesting(
+            macros: testMacros
+        ) {
+            //            InlineSnapshotTesting.isRecording = true
+            //            isRecording: true
```

Comment:
Delete these commented lines, or just uncomment them if necessary and allow people to change the boolean instead of commenting/uncommenting (looks like the second line isn't valid)

Reply:
This is a habit I unknowingly picked from [them](https://github.com/pointfreeco/swift-composable-architecture/blob/main/Tests/ComposableArchitectureMacrosTests/MacroBaseTestCase.swift). I am going to opt to setting it to false unless you have a preference. (Just to serve as living documentation that you can re-record all the test this way). Not sure if they usually comment/uncomment for keyboard shortcut ease or since it's a global variable so setting an actual value could be ambiguous if you do it in several places 🤷

Reply:
Ah yeah makes sense, I think the uncommented boolean toggling way is a bit cleaner but I can see why people would do it the other way too.